docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
<p>This is kind of picky, if we put MAAP Gitlab under notes for MAAP users, I think we should link the MAAP Github as well. </p><p> </p><p>The generic Github link might go under the first bulleted list and generic Gitlab under the other well-known repositories.</p><p> </p><p>What do you think? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
That's a good idea! Before I adjust the links and wording under notes for MAAP users, @wildintellect users are allowed to create repos within the MAAP Gitlab and MAAP Github?

Reply:
MAAP users do not have permissions to add repos to github.com/maap-project this is very different from the MAAP gitlab. If using GitHub it would be recommended that users create a new Org for their research project.

Reply:
Gitlab yes
Github no, that has to be under their own Org or User (Org recommended for collaborative science). the MAAP-Project Org is reserved for "official" uses, users can contribute to those repos like any other open source project.

Reply:
Thanks! Edited to adjust the Github/Gitlab stuff, let me know if anything need to be adjusted.